index.html

```diff
@@ -3100,7 +3100,7 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
           <p data-lang="zh-hant">與內文採相同文字尺寸，字體改為黑體或者楷體。</p>
         </li>
         <li>
-          <p data-lang="en">Set the run-in heading in a character size not smaller than the main text and use Hei or Song.</p>
+          <p data-lang="en">Set the run-in heading in a character size not smaller than the main text and use Hei or Kai.</p>
```

Comment:
Would you please fix the "Song" in the previous sentence (in the same character size case) too?

Reply:
Will make the change.